### PR TITLE
Frontend: Improved readability of Game component

### LINKED ISSUE #29: `dropOnRow` is really confusing
`dropOnRow` on **frontend/src/components/Game.vue** is really confusing and requires an IQ of over 9000 to fully understand.

### PR DESCRIPTION
**Pull request by kenan-rhoton**
I'm hoping this is enough to resolve #29 🙃

### CODE REVIEW

File: frontend/src/components/Game.vue [hunk 1]
```diff
@@ -58,21 +58,14 @@ export default {
...
     gameReq: function() {
-      return (
-        "http://backend:3000/games/" +
-        this.$route.params.gameID +
-        "/player/" +
-        this.$route.params.playerID
-      );
+      const gameID = this.$route.params.gameID;
+      const playerID = this.$route.params.playerID;
+      return `http://backend:3000/games/${gameID}/player/${playerID}`;
```
Comment (Masclins) [inline frontend/src/components/Game.vue]: This is way clearer now.

File: frontend/src/components/Game.vue [hunk 2]
```diff
@@ -58,21 +58,14 @@ export default {
...
     },
     updateGame: async function() {
-      try {
-        const req = this.gameReq();
```
Comment (Masclins) [inline frontend/src/components/Game.vue]: Why removing the try/catch? Or why was it here the first time?

Comment (kenan-rhoton) [reply frontend/src/components/Game.vue]: it was there to catch the exception possibly thrown by a connection error in axios.get... but then I changed the catch to just rethrow the error... and then I realized, if I'm just rethrowing the error, why is the `try` even there?

File: frontend/src/components/Game.vue [hunk 3]
```diff
@@ -84,27 +77,24 @@ export default {
       ev.preventDefault();
       const req = this.gameReq();
       const data = ev.dataTransfer.getData("handIndex");
-      const sentData = { index: data, row: ev.target.getAttribute("rownum") };
-      axios.post(req, sentData);
+      const playData = {
```
Comment (Masclins) [inline frontend/src/components/Game.vue]: More readable.
I like using more comprehensive names.

File: frontend/src/components/Game.vue [hunk 4]
```diff
@@ -84,27 +77,24 @@ export default {
...
-      return (
-        loc.protocol +
-        "//" +
-        loc.hostname +
-        ":" +
-        loc.port +
-        "/#/join-game/" +
-        this.$route.params.gameID
-      );
+      const l = window.location;
```
Comment (Masclins) [inline frontend/src/components/Game.vue]: How is `l` better than `loc`?

Comment (kenan-rhoton) [reply frontend/src/components/Game.vue]: it's literally used only two lines down and I only define the constant to make the code shorter and fit in one line so it's easy to read the format

**Pull request merged**